src/rAF/update.ts

```diff
@@ -55,12 +55,17 @@ export function update<C>({
       }
 
       // Conditions for stopping the physics animation.
+      // If no repeat is specified and we've reached the stopping condition...
       if (
-        (!element.repeat && checkStoppingCondition(element)) ||
-        element.state.repeatCount === element.repeat
+        (typeof element.repeat !== 'number' || element.repeat <= 0) &&
```

Comment:
This will guard against:

- `repeat` specified as a `string`
- `repeat` <= 0, which should just come to a stop after the initial animation completes.